dallinger/experiment.py

```diff
@@ -96,6 +96,12 @@ def __init__(self, session=None):
             "Transformation": Transformation,
         }
 
+        #: dictionary, the properties of this experiment that are exposed
+        #: to the public over an AJAX call
+        if not hasattr(self, 'public_properties'):
+            # Guard against subclasses replacing this with a @property
+            self.public_properties = {}
```

Comment:
Note, in the `waiting-room` branch the experiment_property endpoint is no longer used to get the quorum, so this change could happen separately.